app/src/routes/routesMap.js

```diff
@@ -80,8 +80,8 @@ import { TEST_ITEM_PAGE } from 'controllers/pages/constants';
 import { fetchTestItemsAction, setLevelAction } from 'controllers/testItem';
 import { fetchFiltersPageAction } from 'controllers/filter';
 import { fetchMembersAction } from 'controllers/members';
-import { fetchProjectDataAction } from 'controllers/administrate';
-import { fetchAllUsersAction } from 'controllers/administrate/allUsers';
+import { fetchProjectDataAction } from 'controllers/administrate/actionCreators';
```

Comment:
Circular dependencies again? Is there another way of resolving this?

Reply:
Fixed